templates/beego/files/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"github.com/astaxie/beego"
+	"github.com/platformsh/template-beego/controllers"
+	_ "github.com/platformsh/template-beego/routers"
+)
+
+func getPort() string {
+	var c controllers.MainController
+	c.Prepare()
+	return c.Config.Port()
+}
```

Comment:
Clever!  I don't know if it's too clever, but definitely clever. :smile:

Reply:
I don't know either, but it works!